.circleci/config.yml

```diff
@@ -0,0 +1,81 @@
+version: '2.1'
+orbs:
+  slack: circleci/slack@4.1
+  jq: circleci/jq@2.2.0
+jobs:
+  talisman:
+    docker:
+      - image: rameinstein/talisman:talisman
+        auth:
+          username: rameinstein  # can specify string literal values
+          password: ygvBHU@3690
```

Comment:
I don't understand what's going on here... Did you include your real password in a PR to my project? Please **stop** making PRs trying to set this up with CI, unless your change is broadly applicable to all other users.